sdb/commands/zfs/dbuf.py

```diff
@@ -90,19 +92,29 @@ def pretty_print(self, dbufs):
                 Dbuf.ObjsetName(dbuf.db_objset)))
 
     def argfilter(self, db: drgn.Object) -> bool:
-        if self.args.object and db.db.db_object != self.args.object:
+        if self.args.object is not None and db.db.db_object != self.args.object:
```

Comment:
Could you add a comment here mentioning that you explicitly check if it `is not None` because `0` also evaluates to false?